languagetool-http-client/src/main/java/org/languagetool/remote/SentenceAnnotator.java

```diff
@@ -115,14 +115,11 @@ private static void runAnnotation(AnnotatorConfig cfg) throws IOException, NoSuc
         break;
       }
       String[] partsLine = line.split("\t");
-      String sentence;
-      String sentenceHash = "Undefined";
+      String sentence = partsLine[0];
```

Comment:
In the new GEMA files the `SentenceID` is the *first* column, not the second. So it should be the other way around.

Reply:
This is the input, not the output.

According to https://github.com/languagetooler-gmbh/languagetool-premium/issues/6978#issuecomment-1893440159
> SentAnn is able to accept, as input, a TSV with an ID as the second column;

Reply:
Oh, blimey, well, I misspoke in that comment... As you can see from the [actual files](https://github.com/languagetooler-gmbh/GEMA/blob/main/Annotations/PT/Source/24A.tsv) I posted last week, the ID is in the first column. Sorry about the confusion.

Reply:
As for the output, the [ERMA code](https://github.com/languagetooler-gmbh/errant-for-gema/pull/57/files) assumes the `SentenceID` is the second column, after the hash.